src/pynwb/globals.py

```diff
@@ -0,0 +1,4 @@
+"""Definitions of global values used across the project."""
+
+CORE_NAMESPACE = 'core'
+__core_ns_file_name = 'nwb.namespace.yaml'
```

Comment:
Alternative: also not unusual to call such a collection `consts` such as in https://github.com/dandi/dandi-cli/blob/master/dandi/consts.py

Basically just a collection of all the hand-typed or hard-coded things local to the package